modules/administration/pages/image-management.adoc

```diff
@@ -541,7 +510,13 @@ By default, the image store is located at [path]``/srv/www/os-images``.
 ====
 Image stores for Kiwi build type, used to build system, virtual, and other images, are not supported yet.
 
+ifeval::[{suma-content} == true]
 Images are always stored in [path]``/srv/www/os-images/<organization id>`` and are accessible via HTTP/HTTPS [url]``https://<susemanager_host>/os-images/<organization id>``.
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+Images are always stored in [path]``/srv/www/os-images/<organization id>`` and are accessible via HTTP/HTTPS [url]``https://<uyuni_host>/os-images/<organization id>``.
+endif::[]
```

Comment:
I'm wondering whether we'd need an empty line after the `endif` and before the `====` line?

Reply:
My understanding is that we do.